src/main/resources/org/jenkinsci/plugins/zap/ZAPDriver/config.jelly

```diff
@@ -128,6 +128,9 @@ SOFTWARE.
             <f:entry title="${%Logged in Indicator}" field="loggedInIndicator">
               <f:textbox clazz="required" />
             </f:entry>
+            <f:entry title="${%Logged out Indicator}" field="loggedOutIndicator">
+              <f:textbox clazz="required" />
```

Comment:
i added it because in a project i needed it.(because there were no logged in indicator)
but you can still build a project event if it is not defined. but you're write, i should have put it in an optional block